src/cursor.c

```diff
@@ -881,6 +881,28 @@ cursor_button(struct wl_listener *listener, void *data)
 	}
 }
 
+static int
+compare_delta(const struct wlr_pointer_axis_event *event, double *accum)
+{
+	const double SCROLL_THRESHOLD = 10.0;
```

Comment:
A comment explaining this value would be good, e.g. where does it come from / why exactly `10.0`.

Reply:
Sadly this value was invented whole cloth on the basis of what "feels right". I'll add a comment to that effect.

I have a vague memory of there being a similar 10 hard-coded somewhere in the Gdk/Gtk stack, but wasn't able to find it on a cursory search. I'll look again when I get a chance.

Reply:
I've tracked down a place explaining the history of the 10 constant (which is in Mutter, libinput, and Weston, not GTK) and added a comment explaining its origin.